package.json

```diff
@@ -17,8 +17,8 @@
     "@types/node": "10.17.27",
     "aws-cdk-lib": "2.134.0",
     "constructs": "10.1.67",
-    "jest": "^26.4.2",
-    "ts-jest": "^26.2.0",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.2",
```

Comment:
This is major version bump. Is it required? Might be breaking stuff

Reply:
This is created by the dependabot, so I block it as it failed the runs.